apps/backend/src/app/cloudinary/cloudinary.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common';
+import { CloudinaryService } from './cloudinary.service';
+import { HotelController } from '../hotel/hotel.controller';
+import { ConfigModule } from '@nestjs/config';
+import cloudinaryConfig from './cloudinary.config';
+
+@Module({ 
+    imports:[ConfigModule.forRoot({ load: [cloudinaryConfig] })],
```

Comment:
let try `ConfigModule.forFeature(() =>{...})`, because cloudinary is new feature for backend